scenes/cslib_number.txt

```diff
@@ -118,6 +118,52 @@
 *goto min_loop
 
 
+*comment MAX_STAT
```

Comment:
I'm thinking MAX_VAR and MIN_VAR is more general? Or does STAT help imply that it only works for globals? Hmm.

Reply:
Also makes me wonder about lowest/highest vs smallest/largest 😩

Reply:
STAT b/c you need global (due to gosub_scene)
MAX is the largest, what else?